ecosystem/sep-0024.md

```diff
@@ -73,6 +74,54 @@ To support the exchange of non-equivalent assets using this protocol, the anchor
 
 Finally, anchors should populate the optional attributes listed in the `GET /transaction(s)` endpoints related to non-equivalent asset transactions, specifically `amount_in_asset`, `amount_fee_asset`, and `amount_out_asset`. This allows client applications can relay this information, in addition to the amounts specified in the associated attributes, back to their users.
 
+Alternatively, the Anchor must support `quote_id` in the `POST /transactions/deposit/interactive` and `POST /transactions/withdraw/interactive`, and the [SEP-38] Anchor RFQ API that enables the Anchor to provide quotes to the Wallet. 
+
+### Asset Conversions Using the Market Rate
+
+To convert at the market rate, clients get an [indicative quote](https://www.investopedia.com/terms/i/indicativequote.asp) with [SEP-38] and then proceed to calling the [`GET /deposit-exchange`](#deposit-exchange) or [`GET /withdraw-exchange`](#withdraw-exchange) endpoints. Market rate conversions executed in this way do not provide a guarantee of the final price to be paid for the asset. Here is how this kind of conversion should happen in deposits and withdrawals for an asset pair like `BRL <> USDC`:
+
+#### Deposit:
+
+Considering the user will be making a bank deposit of fiat `BRL` in exchange for `USDC` in the Stellar network:
+
+1. The Wallet checks the assets' market conversion rate with [`SEP-38 GET /price`](sep-0038.md#get-price) and shows the estimated price to the user.
+1. The Wallet initiates a [`POST /transactions/deposit/interactive`](#deposit) using the parameter `asset_code`.
+1. The user proceeds with the off-chain deposit, sending `BRL` to the Anchor's bank account.
+1. Upon receiving the off-chain deposit funds, the Anchor converts it to `USDC` using the market rate and sends the resulting amount to the user's Stellar account.
+
+#### Withdraw:
+
+Considering the user will be making a Stellar payment of `USDC` in exchange for withdrawing `BRL` in the bank account:
+
+1. The Wallet checks the assets' market conversion rate with [`SEP-38 GET /price`](sep-0038.md#get-price) and shows the estimated price to the user.
+1. The Wallet initiates a [`POST /transactions/withdraw/interactive`](#withdraw) using the parameter `asset_code`.
+1. The user proceeds with the on-chain payment, sending `USDC` to the Anchor's Stellar account.
+1. Upon receiving the on-chain withdraw funds, the Anchor converts it to `BRL` using the market rate and sends the resulting amount to the user's bank account.
+
+### Asset Conversions Using a Firm Quote
+
+With firm quotes, users are guaranteed an asset price they will pay if they submit the payment before the quote expiration. Here is how the conversion should happen in deposits and withdrawals for an asset pair like `USDC <> BRL`:
```

Comment:
nit: the quote is honored only if the payment was _received_ by the anchor before the quote's expiration. For off-chain payments this is important since they can take some time.